Makefile

```diff
@@ -61,12 +62,15 @@ gen-manifests: bin/helm
 		CALICO_VERSION=$(CALICO_VERSION) \
 		./generate.sh
 
-# Get operator CRDs from the operator repo, OPERATOR_BRANCH_NAME must be set
-get-operator-crds: var-require-all-OPERATOR_BRANCH_NAME
+# Get operator CRDs from the operator repo, OPERATOR_BRANCH must be set
+get-operator-crds: var-require-all-OPERATOR_BRANCH
+	@echo ================================================================
+	@echo === Pulling new operator CRDs from branch ${OPERATOR_BRANCH} ===
+	@echo ================================================================
 	cd ./charts/tigera-operator/crds/ && \
-	for file in operator.tigera.io_*.yaml; do echo "downloading $$file from operator repo" && curl -fsSL https://raw.githubusercontent.com/tigera/operator/${OPERATOR_BRANCH_NAME}/pkg/crds/operator/$${file%_crd.yaml}.yaml -o $${file}; done
+	for file in operator.tigera.io_*.yaml; do echo "downloading $$file from operator repo" && curl -fsSL https://raw.githubusercontent.com/tigera/operator/${OPERATOR_BRANCH}/pkg/crds/operator/$${file%_crd.yaml}.yaml -o $${file}; done
```

Comment:
```suggestion
	for file in operator.tigera.io_*.yaml; do echo "downloading $$file from operator repo" && curl -fsSL https://raw.githubusercontent.com/tigera/operator/$(OPERATOR_BRANCH)/pkg/crds/operator/$${file%_crd.yaml}.yaml -o $${file}; done
```

Reply:
we need `$()` to refer to the Makefile var, not an env var (which was also incorrect, it would have needed to have `$${}`)

Reply:
so @danudey told me I'm wrong and `${}` is supported (TIL, thanks!), but I still believe we should try to stick to `$()`

Reply:
I can swap (although I will note that these predate my involvement in this area of our Makefile!)